packages/in-memory-live-query-store/src/InMemoryLiveQueryStore.ts

```diff
@@ -8,6 +8,7 @@ import {
   defaultFieldResolver,
   TypeInfo,
 } from "graphql";
+import { getArgumentValues } from "graphql/execution/values.js";
```

Comment:
You can import this from `@graphql-tools/utils` because deeper imports from graphql-js might be inconsistent